src/main/java/org/folio/circulation/domain/notice/schedule/RequestScheduledNoticeService.java

```diff
@@ -80,7 +80,7 @@ public Result<CheckInContext> rescheduleRequestNotices(CheckInContext context) {
     return succeeded(context);
   }
 
-  private void rescheduleRequestNotices(Request request) {
+  public void rescheduleRequestNotices(Request request) {
```

Comment:
Can we use `rescheduleRequestNotices(RequestAndRelatedRecords relatedRecords)` instead and keep this method private? I get it that we didn't have `relatedRecords` in the `UpdateRequestQueue`, but we will in the resource.